### PR TITLE
House: Add house description

### PR DESCRIPTION
**Pull request by ronyyosef**

### CODE REVIEW

**Review by LiorKGOW — COMMENTED** [review]
Why does house need description?

File: expenses/migrations/0001_initial.py [hunk 1]
```diff
@@ -1,38 +1,22 @@
...
-                            ("Vacations", "Vacations"),
-                            ("Other", "Other"),
-                        ],
-                        default="Other",
-                        max_length=32,
-                    ),
-                ),
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('description', models.TextField(default='', max_length=250)),
+                ('category', models.CharField(choices=[('Rent', 'Rent'), ('Mortgage', 'Mortgage'), ('bills', 'Bills'), ('Transportation', 'Transportation'), ('Clothing', 'Clothing'), ('Healthcare', 'Healthcare'), ('Food', 'Food'), ('Insurance', 'Insurance'), ('Kids', 'Kids'), ('Culture', 'Culture'), ('Vacations', 'Vacations'), ('Other', 'Other')], default='Other', max_length=32)),
```
Comment (LiorKGOW) [inline expenses/migrations/0001_initial.py:19]: Like #50 this line doesn't pass the flake8 maxline check

Comment (LiorKGOW) [reply expenses/migrations/0001_initial.py:19]: Resolved in #50 
ignore this comment

**Comment by ronyyosef** [comment]
> Why does the house need a description?

We decide to remove the Story app and add a description filed to the house

**Comment by adiabramovitch** [comment]
There are some files that are not related to the PR title, if you don't want to use several commits which is fine you should open a different PR for "prettifying the code"

**Comment by shlior7** [comment]
the test fail because it not rebase with the main branch, i suggest edit the first commit using rebase and then fetch and rebase to origin/main

**Comment by ronyyosef** [comment]
Ready to merge

**Review by Yoavsc129 — APPROVED** [review]

File: house/migrations/0001_initial.py [hunk 2]
```diff
@@ -1,68 +1,51 @@
...
-                ('parent_profession_1', models.CharField(max_length=50, choices=jobs)),
-                ('parent_profession_2', models.CharField(max_length=50, choices=jobs)),
-                ('income', models.IntegerField(blank=True, null=True)),
-                ('children', models.IntegerField(blank=True, null=True)),
+                ('parent_profession_1', models.CharField(choices=[('Teacher', 'Teacher'), ('Student', 'Student'), ('Programmer', 'Programmer'), ('Artist', 'Artist'), ('Manager', 'Manager'), ('Army', 'Army'), ('Police', 'Police'), ('Doctor', 'Doctor'), ('Vet', 'Vet'), ('Nurse', 'Nurse'), ('Technichian', 'Technichian'), ('Cleaner', 'Cleaner'), ('Other', 'Other'), ('Unemployed', 'Unemployed')], default='Other', max_length=50)),
+                ('parent_profession_2', models.CharField(choices=[('Teacher', 'Teacher'), ('Student', 'Student'), ('Programmer', 'Programmer'), ('Artist', 'Artist'), ('Manager', 'Manager'), ('Army', 'Army'), ('Police', 'Police'), ('Doctor', 'Doctor'), ('Vet', 'Vet'), ('Nurse', 'Nurse'), ('Technichian', 'Technichian'), ('Cleaner', 'Cleaner'), ('Other', 'Other'), ('Unemployed', 'Unemployed')], default='Other', max_length=50)),
```
[the remark this answers is not in the document]
Comment (adiabramovitch) [reply house/migrations/0001_initial.py:38]: ?

**Comment by ronyyosef** [comment]
Ready for merge

**Review by Yoavsc129 — APPROVED** [review]

**Comment by adiabramovitch** [comment]
You resolved my comment and did not comment about it.

File: house/migrations/0001_initial.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (ronyyosef) [reply house/migrations/0001_initial.py:38]: Screenshot_20220411-102503_Chrome
Was in pending, sory

**Review by LiorKGOW — APPROVED** [review]

**Pull request merged**